src/components/BaseCommentForm.js

```diff
@@ -13,7 +13,7 @@ import forEach from 'lodash/forEach';
 export class BaseCommentForm extends React.Component {
   constructor(props, context) {
     super(props, context);
-    this.state = {collapsed: true, commentText: "", nickname: "", imageTooBig: false, images: []};
+    this.state = {collapsed: true, commentText: "", nickname: props.defaultNickname || 'Mörö', imageTooBig: false, images: []};
```

Comment:
I'm afraid the default nickname in absence of props will just have to be empty :(

Reply:
Oh, this is embarrassing leftover from testing :D Will be removed